app/views/admin/layouts/_area.html.erb

```diff
@@ -0,0 +1,12 @@
+<% x = area.coordinate.nil? ? 0 : area.coordinate.x %>
+<% y = area.coordinate.nil? ? 0 : area.coordinate.y %>
+<% type = area.class.name.downcase %>
+<div class="<%= type %> draggable"
+     data-toggle="tooltip"
+     title="<%= area.name %>"
+     style="transform: translate(<%= x %>px, <%= y %>px)"
+     rails-id="<%= area.id %>"
+     rails-type="<%= type %>"
```

Comment:
Since you're not using `type` anywhere else, you can just inline `rails-type="<%= area.class.name.downcase %>"` as to keep everything nice and tight

Reply:
`type` is used to define a class as well

Reply:
In the next PR I made a function in the model for `type` which returns `.class.name.downcase`. I find it a lot cleaner.

I also replaced this partial with a helper called `area_tag`. I personally find it cleaner.